client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/object/CustomTypeToOasConverter.java

```diff
@@ -0,0 +1,6 @@
+package org.evomaster.client.java.instrumentation.object;
+
+public abstract class CustomTypeToOasConverter {
```

Comment:
add some JavaDoc for this class. for example, what is `Oas`?

Reply:
OAS refers to Open API Specification.

Reply:
Documentation added